debian/gen-missing-architecture-fixtures.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+set -eux
+
+SRCDIR=$(readlink -f "$(dirname "$0")")
+TMPDIR=$(mktemp -d)
+OUTPUTDIR=$(realpath "${SRCDIR}/../$1")
+
+trap 'rm -rf "${TMPDIR}"' EXIT
+
+cd "${TMPDIR}"
+
+mkdir asgard
+(
+  cd asgard
+
+  for CTL in "${SRCDIR}"/asgard/*.ctl
+  do
+    equivs-build --arch ppc64 "${CTL}"
+  done
+)
+
+mkdir asgard_udebs
+(
+  cd asgard_udebs
+
+  for CTL in "${SRCDIR}"/asgard_udebs/*.ctl
+  do
+    equivs-build --arch ppc64 "${CTL}"
+  done
+  for DEB in *.deb
+  do
+    mv "$DEB" "${DEB%deb}udeb"
+  done
+)
+
+mkdir jotunheimr
+(
+  cd jotunheimr
+
+  for CTL in "${SRCDIR}"/jotunheimr/*.ctl
+  do
+    equivs-build --arch armeb "${CTL}"
+  done
+)
+
+cp -a "${SRCDIR}/conf" .
+reprepro -C asgard includeudeb ragnarok asgard_udebs/*.udeb
+reprepro -C asgard includedeb ragnarok asgard/*.deb
+reprepro -C asgard includedeb ragnarok jotunheimr/*.deb
+reprepro -C jotunheimr includedeb ragnarok jotunheimr/*.deb
+
+# No Packages files at all for architecture armeb
+rm -r dists/ragnarok/asgard/binary-armeb
+rm -r dists/ragnarok/jotunheimr/binary-armeb
```

Comment:
This is what makes this repository special?

What is the expected behaviour here? Failing, because the files are really expected?

Reply:
Yes, pulp would throw an Error for architectures with missing Packages file, please also see https://github.com/pulp/pulp_deb/pull/226/ for reference.